### PR TITLE
style: singleLineMap component

### PR DESCRIPTION
**Pull request by zushar**
# Description
<improve space on the sreen for bigger map>

## screenshots
Screenshot 2024-05-01 120754

### CODE REVIEW

File: src/pages/singleLineMap/index.tsx [hunk 1]
```diff
@@ -127,75 +125,66 @@ const SingleLineMapPage = () => {
           videoUrl="https://www.youtube-nocookie.com/embed/bXg50_j_hTA?si=inyvqDylStvgNRA6&amp;start=93"
         />
       </Title>
-      <Space direction="vertical" size="middle" style={{ marginBottom: '22px' }}>
-        <Alert message={t('realtime_map_page_description')} type="info" />
```
Comment (NoamGaash) [inline src/pages/singleLineMap/index.tsx:131]: don't we want it?

Comment (zushar) [reply src/pages/singleLineMap/index.tsx:131]: There is a YouTube video for that reason no? Except for that I think the page is pretty self-explanatory

Comment (NoamGaash) [reply src/pages/singleLineMap/index.tsx:131]: you're right, so if this text isn't used anywhere else, perhaps we can remove it from the translation files?

Comment (NoamGaash) [reply src/pages/singleLineMap/index.tsx:131]: Or better - maybe we can add this paragraph to the YouTube modal.
This paragraph, unlike the video, can be translated to English when needed

Comment (zushar) [reply src/pages/singleLineMap/index.tsx:131]: That makes perfect sense I will work on that

Comment (NoamGaash) [reply src/pages/singleLineMap/index.tsx:131]: Thank you! :clap:

**Comment by zushar** [comment]
Screenshot 2024-05-03 105611
this is a screenshot of youtubModel which includes a description for the page

File: src/pages/singleLineMap/index.tsx [hunk 2]
```diff
@@ -123,79 +121,70 @@ const SingleLineMapPage = () => {
         {t('singleline_map_page_title')}
         <InfoYoutubeModal
           label={t('open_video_about_this_page')}
-          title={t('youtube_modal_info_title')}
+          title={t('realtime_map_page_description')}
```
Comment (NoamGaash) [inline src/pages/singleLineMap/index.tsx:124]: I think it's not the description of the right page, and perhaps we should consider includith both title and paragraph in the modal

**Review by NoamGaash — APPROVED** [review]
Thank you! that's a good improvement, feel free to merge when you'll feel it's ready

**Comment by NoamGaash** [comment]
@all-contributors please add @zushar as code contributor

Thanks for the great work! the page does look much better :partying_face:

**Pull request merged**